### PR TITLE
docs: updating cli upgrade versions and linting

### PR DESCRIPTION
**Pull request by msdolbey**
## Jira Ticket

N/A

## Description of changes being made
This was mainly upgrading the version for the Konvoy CLI upgrade path.

It looks like there's a lot of changes, but that's mostly linting on the pages I touched.

### Preview

You can preview the docs build using the following URL, adding the PR # where specified:
http://docs-d2iq-com-pr-4137.s3-website-us-west-2.amazonaws.com/

## Checklist

- [ ] Test all commands and procedures, if applicable.
- [ ] Update all links if you are moving a page.
- [ ] Add release date to Release Notes page in the following format: <Package> was released on <Day>, <Month> <Year> Example: `Mesosphere® DC/OS™ 2.1.0 was released on 9, June 2020`

See the [contribution guidelines](https://github.com/mesosphere/dcos-docs-site/blob/main/CONTRIBUTING.md) for more information.

### CODE REVIEW

File: pages/dkp/konvoy/1.8/upgrade/upgrade-cli/index.md [hunk 1]
```diff
@@ -43,32 +43,32 @@ If you are using a [private Docker registry][docker_registry] for your clusters,
...
 
-### Upgrading Konvoy from v1.7.x to v1.8.4
+### Upgrading Konvoy from v1.7.x to v1.8.5
 
 **You must modify your `cluster.yaml` with these changes when upgrading from a previous Konvoy version:**
 
 It is recommended to upgrade to the newest supported version of Kubernetes. Set `spec.kubernetes.version: 1.20.13` for of kind: ClusterConfiguration.
 
-The version of Calico got bumped in Konvoy v1.8.4, from Calico v3.17.3 to v3.20.2. You will want to change this in `spec.containerNetworking.calico.version: v3.17.3` for of kind: ClusterConfiguration.
+The version of Calico got bumped in Konvoy v1.8.5, from Calico v3.17.3 to v3.20.2. You will want to change this in `spec.containerNetworking.calico.version: v3.17.3` for of kind: ClusterConfiguration.
```
Comment (cneth) [inline pages/dkp/konvoy/1.8/upgrade/upgrade-cli/index.md:58]: I believe this is incorrect, the Calico bump happened in 1.8.4, not 1.8.5.  

```suggestion
The version of Calico was bumped in Konvoy v1.8.4, from Calico v3.17.3 to v3.20.2.  When upgrading, set `spec.containerNetworking.calico.version: v3.20.2` for kind: ClusterConfiguration.
```

File: pages/dkp/konvoy/1.8/upgrade/upgrade-cli/index.md [hunk 2]
```diff
@@ -43,32 +43,32 @@ If you are using a [private Docker registry][docker_registry] for your clusters,
...
 
-The version of Kubernetes Base Addons changed if you use KBA, so you need to change your `configVersion` for your `configRepository`: `https://github.com/mesosphere/kubernetes-base-addons` to be `spec.addons.configVersion: stable-1.20-4.3.0` for of kind: ClusterConfiguration.
+The version of Kubernetes Base Addons changed if you use KBA, so you need to change your `configVersion` for your `configRepository`: `https://github.com/mesosphere/kubernetes-base-addons` to be `spec.addons.configVersion: stable-1.20-4.4.0` for of kind: ClusterConfiguration.
```
Comment (cneth) [inline pages/dkp/konvoy/1.8/upgrade/upgrade-cli/index.md:60]:
```suggestion
The version of Kubernetes Base Addons changed if you use KBA, so you need to change your `configVersion` for your `configRepository`: `https://github.com/mesosphere/kubernetes-base-addons` to be `spec.addons.configVersion: stable-1.20-4.4.0` for kind: ClusterConfiguration.
```

File: pages/dkp/konvoy/1.8/upgrade/upgrade-cli/index.md [hunk 3]
```diff
@@ -43,32 +43,32 @@ If you are using a [private Docker registry][docker_registry] for your clusters,
...
 
-If you use Kommander, you need to change the `configVersion` for your `configRepository`: `https://github.com/mesosphere/kubeaddons-kommander` to be `spec.addons.configVersion: stable-1.20-1.4.3` for of kind: ClusterConfiguration.
+If you use Kommander, you need to change the `configVersion` for your `configRepository`: `https://github.com/mesosphere/kubeaddons-kommander` to be `spec.addons.configVersion: stable-1.20-1.4.4` for of kind: ClusterConfiguration.
```
Comment (cneth) [inline pages/dkp/konvoy/1.8/upgrade/upgrade-cli/index.md:62]:
```suggestion
If you use Kommander, you need to change the `configVersion` for your `configRepository`: `https://github.com/mesosphere/kubeaddons-kommander` to be `spec.addons.configVersion: stable-1.20-1.4.4` for kind: ClusterConfiguration.
```

File: pages/dkp/konvoy/1.8/upgrade/upgrade-cli/index.md [hunk 4]
```diff
@@ -43,32 +43,32 @@ If you are using a [private Docker registry][docker_registry] for your clusters,
...
 
 If you have Dispatch enabled, you need to change the `configVersion` for your `configRepository`: `https://github.com/mesosphere/kubeaddons-dispatch` to be `configVersion: stable-1.20-1.4.6` for of kind: ClusterConfiguration
```
Comment (cneth) [inline pages/dkp/konvoy/1.8/upgrade/upgrade-cli/index.md:64]:
```suggestion
If you have Dispatch enabled, you need to change the `configVersion` for your `configRepository`: `https://github.com/mesosphere/kubeaddons-dispatch` to be `configVersion: stable-1.20-1.4.6` for kind: ClusterConfiguration
```

File: pages/dkp/konvoy/1.8/upgrade/upgrade-cli/index.md [hunk 5]
```diff
@@ -43,32 +43,32 @@ If you are using a [private Docker registry][docker_registry] for your clusters,
...
 ```
 
 After the upgrade command completes, you can start using the new Konvoy version.
 
-### Upgrading Konvoy from v1.7.x to v1.8.4
+### Upgrading Konvoy from v1.7.x to v1.8.5
 
 **You must modify your `cluster.yaml` with these changes when upgrading from a previous Konvoy version:**
 
 It is recommended to upgrade to the newest supported version of Kubernetes. Set `spec.kubernetes.version: 1.20.13` for of kind: ClusterConfiguration.
```
Comment (cneth) [inline pages/dkp/konvoy/1.8/upgrade/upgrade-cli/index.md:56]: I've never noticed this before, but the phrasing 'for of kind' in all these items is very strange and awkward.

```suggestion
It is recommended to upgrade to the newest supported version of Kubernetes. Set `spec.kubernetes.version: 1.20.13` for kind: ClusterConfiguration.
```

File: pages/dkp/konvoy/1.8/upgrade/upgrade-cli/index.md [hunk 6]
```diff
@@ -43,32 +43,32 @@ If you are using a [private Docker registry][docker_registry] for your clusters,
...
 
-If you use Kommander, you need to change the `configVersion` for your `configRepository`: `https://github.com/mesosphere/kubeaddons-kommander` to be `spec.addons.configVersion: stable-1.20-1.4.3` for of kind: ClusterConfiguration.
+If you use Kommander, you need to change the `configVersion` for your `configRepository`: `https://github.com/mesosphere/kubeaddons-kommander` to be `spec.addons.configVersion: stable-1.20-1.4.4` for of kind: ClusterConfiguration.
 
 If you have Dispatch enabled, you need to change the `configVersion` for your `configRepository`: `https://github.com/mesosphere/kubeaddons-dispatch` to be `configVersion: stable-1.20-1.4.6` for of kind: ClusterConfiguration
 
-The version of Konvoy is now `v1.8.4`, set `spec.version: v1.8.4` for both of kind: ClusterProvisioner and of kind: ClusterConfiguration.
+The version of Konvoy is now `v1.8.5`, set `spec.version: v1.8.5` for both of kind: ClusterProvisioner and of kind: ClusterConfiguration.
```
Comment (cneth) [inline pages/dkp/konvoy/1.8/upgrade/upgrade-cli/index.md:66]:
```suggestion
The version of Konvoy is now `v1.8.5`, set `spec.version: v1.8.5` for both kind: ClusterProvisioner and kind: ClusterConfiguration.
```

**Review by cneth — APPROVED** [review]

**Review by mwrightd2 — APPROVED** [review]

File: pages/dkp/konvoy/1.8/upgrade/upgrade-cli/index.md (hunk 1 above)
Comment (msdolbey) [reply pages/dkp/konvoy/1.8/upgrade/upgrade-cli/index.md:58]: Good catch with that wording @cneth

**Comment by msdolbey** [comment]
Thanks all! Merging!

**Pull request merged**